lib/screens/app/wallet/dashboard.dart

```diff
@@ -408,7 +417,7 @@ class _DashboardState extends State<Dashboard> {
                             children: <Widget>[
                               Container(
                                 height: 16,
-                                width: 320,
+                                //width: 320,
```

Comment:
needed?

Reply:
remvoed

Reply:
Note: This caused renderflex exceptions on the iPhone 11 Pro Simulator.